templates/debug.php

```diff
@@ -714,9 +750,19 @@ function stopCountdownManually() {
         </table>
     <?php endif ?>
 <?php endforeach ?>
+<?php
+    $page_params = array(
+        'title_tag' => 'h2',
+        'display_toggle' => true,
+        'is_open' => false,
+    );
+    fs_require_template( 'debug/scheduled-crons.php', $page_params );

```

Comment:
@DanieleAlessandra We can have some alignment like this:
```
$page_params = array(
    'title_tag'            => 'h2',
    'display_toggle' => true,
    'is_open'            => false,
);

fs_require_template( 'debug/scheduled-crons.php', $page_params );
```

Also, I added an empty line above `fs_require_template()`.

Reply:
Not relevant anymore because final array only has one element